src/components/Menu/MenuItem.tsx

```diff
@@ -43,11 +51,38 @@ export function MenuItem({
   const { menuOpen, toggleMenuOpen } = useContext(LayoutContext);
   const asPathWithoutHash = usePathWithoutHash();
   const [open, setOpen] = useState(false);
+  const ref = useRef(null);
   const children = useMemo(
     () => (hideChildren ? [] : pageNode.children),
     [hideChildren, pageNode.children]
   );
+
+  const setSelectability = () => {
+    const current = ref.current;
+    const items = current?.parentElement?.nextSibling?.children;
+    if (items) {
+      for (const item of items) {
+        const links = item.getElementsByTagName('a');
+        const buttons = item.getElementsByTagName('button');
+        if (links[0].getAttribute('tabIndex') == 0) {
+          for (const link of links) {
+            link.setAttribute('tabIndex', -1);
```

Comment:
It looks like the desired tab behavior only works when opening the link after landing on a page, reloading while already on a page leaves these visible menu links untabbable

https://github.com/user-attachments/assets/bec1f47b-7909-4cc6-aaab-46436d1dd525